Simon.py

```diff
@@ -0,0 +1,282 @@
+# Simon
+import pygame, sys, random, time
+from pygame.locals import *
+
+pygame.init()
+DISPLAY = pygame.display.set_mode((700, 500))
+pygame.display.set_caption('Simon')
+fpsClock = pygame.time.Clock()
+fpsClock.tick(60)
+
+#Colors
+white        = (255, 255, 255)
+black        = (0, 0, 0)
+
+yellow       = (155, 155, 0)
+yellow_light = (255, 255, 0,)
+blue         = (0, 0, 155)
+blue_light   = (0, 0, 255)
+red          = (155, 0, 0)
+red_light    = (255, 0, 0)
+green        = (0, 128, 0)
+green_light  = (0, 255, 0)
+screen_color = black
+
+#Text
+font1 = pygame.font.Font('freesansbold.ttf', 46)
+font2 = pygame.font.Font('freesansbold.ttf', 18)
+font3 = pygame.font.Font('freesansbold.ttf', 22)
+
+titleText = font1.render('Simon', True, black, white)
+titleRect = titleText.get_rect()
+titleRect.center = (350, 100)
+
+introText = font2.render('Press ENTER to begin.', True, white, black)
+introRect = introText.get_rect()
+introRect.center = (350, 350)
+
+directionsText = font2.render('Use the arrow keys to repeat the pattern on the screen.', True, white, black)
+directionsRect  = directionsText.get_rect()
+directionsRect.center = (350, 310)
+
+correctText = font2.render('Correct!', True, white, black)
+correctRect = correctText.get_rect()
+correctRect.center = (350, 420)
+
+incorrectText = font2.render('Incorrect.', True, white, black)
+incorrectRect = incorrectText.get_rect()
+incorrectRect.center = (350, 420)
+
+averageText = font2.render('Your average is ' + str() + '.', True, white, black)
+averageRect = averageText.get_rect()
+averageRect.center = (350, 250)
+
+
+endText1 = font3.render('Do you want to play again?', True, white, black)
+endRect1 = endText1.get_rect()
+endRect1.center = (350, 400)
+
+endText2 = font2.render('If you do, press "y", otherwise close this window or press "esc" to exit the game.',True, white, black)
+endRect2 = endText2.get_rect()
+endRect2.center = (350,435)
+
+
+
+#scoreText = font2.render('Score:', True, white, black)
+#scoreRect = scoreText.get_rect()
+#scoreRect.center = (550, 30)
+
+#intro_pic = 'Users/s.miles1313/Desktop/images.png'
+
+#Buttons
+top_mid    = pygame.Rect(300, 100, 100, 100)
+#yellow_rect = pygame.draw.rect(DISPLAY, yellow, top_mid)
+
+bottom_mid = pygame.Rect(300, 250, 100, 100)
+#blue_rect  = pygame.draw.rect(DISPLAY, blue, bottom_mid)
+
+bottom_right = pygame.Rect(450, 250, 100, 100)
+#red_rect = pygame.draw.rect(DISPLAY, red, bottom_right)
+
+bottom_left  = pygame.Rect(150, 250, 100, 100)
+#green_rect  = pygame.draw.rect(DISPLAY, green, bottom_left)
+
+patternspeed = 450 #milliseconds
+patterndelay = 200
+
+#K_UP                  up arrow
+#K_DOWN                down arrow
+#K_RIGHT               right arrow
+#K_LEFT                left arrow
+
+def main():
```

Comment:
Right now the boxes are flashing too quickly. Would recommend following a process
1. Can you start with the drawing board with black background and displaying the 4 boxes
2. Add to 1, click tracking on the boxes 
3. Add lighting to 1 box randomly